src/deprecations.jl

```diff
@@ -438,3 +438,8 @@ export VTKFile
 function VTKFile(args...)
     throw(DeprecationError("VTKFile(args...)" => "VTKGridFile(args...)"))
 end
+
+# assemble with vector first
+function assemble!(::AbstractAssembler, ::AbstractVector{<:Integer}, ::AbstractVector, ::AbstractMatrix)
```

Comment:
```suggestion
function assemble!(::Union{AbstractAssembler, COOAssembler}, ::AbstractVector{<:Integer}, ::AbstractVector, ::AbstractMatrix)
```
And add test since this is important that users see if they have this error?

Reply:
COOAssembler doesn't support vector assembly.

Reply:
Got confused since this is being added now :)